swift/ql/lib/codeql/swift/elements/HideableElement.qll

```diff
@@ -0,0 +1,14 @@
+private import codeql.swift.generated.HideableElement
+
+class HideableElement extends Generated::HideableElement {
```

Comment:
We really should have QLDoc on this.

Reply:
actually, I'm about to remove this new class, which will also mean no dbscheme change :+1:

Reply:
OK, I'll re-review...